src/features/Search/MobileFilterPicker.tsx

```
@@ -0,0 +1,121 @@
+  clearAll,
+  objectTypes = ['user', 'resources']
+}) => {
+  const selectorRef = useRef<MobileSelectorHandle>(null)
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  return (
+    <MobileSelector
+      displayText={selectedContacts[0]?.displayName ?? displayText}
+      fullscreen
+      bottomSheetChildren={({ open, onClose }) => {
+        useEffect(() => {
```

ESLint

[error] React Hook "useEffect" cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)